packages/contract/script/pay/DeployDaimoPayBridger.s.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+import "forge-std/Script.sol";
+import "openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+import "../../src/pay/DaimoPayBridger.sol";
+import "../Constants.s.sol";
+
+contract DeployDaimoPayBridger is Script {
+    function run() public {
+        vm.startBroadcast();
+
+        // initOwner = daimo.eth
+        address initOwner = 0xEEee8B1371f1664b7C2A8c111D6062b6576fA6f0;
+
+        DaimoPayBridger implementation = new DaimoPayBridger{salt: 0}(
+            initOwner
```

Comment:
move to `init()`

Reply:
(the implementation contract will have no state > owner = 0x0 > can't run any admin functions. this is ok, the proxy contract owns the state)